Sprint #19.adoc

```diff
@@ -29,9 +29,45 @@
 * TASK: make hasAttachment searchable via mailbox-api
 * TASK: make GetMessagesList call the search api
 
+Remark. If you want to call here the search API for Messages you will have to problems :
+ - It returns a list of UID. You may need to add information about mailboxes depending on your implementations choices
+ - As such the returned document is not present (reduce network load) + the index document is not the original mail (text extraction). So you will still need a read to Cassandra after your read to ElasticSearch..
```

Comment:
Ok, I didn't get it clear when I arrived this morning, but after digging into GetMessageList I figured that out. 

So supporting multiple mailboxes will be super easy, except for.... **Sorting** :-p Maybe you need also the scores from ES to sort this.... What are you planning for this ?